pkg/cmd/server/kubernetes/master.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	kapi "k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/client/record"
+	osclient "github.com/openshift/origin/pkg/client"
```

Comment:
@deads2k what would a remote kube HPA controller do if it encountered a non kube resource reference?